lib/auth/screens/auth_screen.dart

```diff
@@ -236,3 +234,18 @@ class _PasswordField extends StatelessWidget {
     return null;
   }
 }
+
+SnackBar snackBar(BuildContext context, String error) {
```

Comment:
snackBar method should be inside State of statefull widget
move this method inside state, make it private and remove `BuildContext` from arguments of this function